src/ui/src/app/deployment/deployment.component.scss

```diff
@@ -1,38 +1,32 @@
 // Import your custom theme
 @import '../../theme.scss';
 
+.deployment-details__header {
+  width: 100%;
+  max-width: 1000px;
```

Comment:
I see this `max-width: 1000px` property in a few CSS files. Can we move it to a common place?